packages/material/test/renderers/MaterialArrayLayout.test.tsx

```diff
@@ -129,4 +170,30 @@ describe('Material array layout', () => {
     // 2 data entries with each having 2 controls
     expect(controls.length).toBe(4);
   });
+
+  it('should generate uischema when options.detail=GENERATE', () => {
+    const store = initJsonFormsStore();
+    const tree = TestUtils.renderIntoDocument(
+      <Provider store={store}>
+        <MaterialArrayLayout schema={schema} uischema={uischemaOptions.generate}/>
+      </Provider>
+    );
+
+    const controls = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    // 2 data entries with each having 2 controls
+    expect(controls.length).toBe(4);
+  });
+
+  it('should use inline options.detail uischema', () => {
+    const store = initJsonFormsStore();
+    const tree = TestUtils.renderIntoDocument(
+      <Provider store={store}>
+        <MaterialArrayLayout schema={schema} uischema={uischemaOptions.generate}/>
+      </Provider>
+    );
+
+    const controls = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    // 1 data entries having 2 controls
+    expect(controls.length).toBe(4);
```

Comment:
Does the comment match the expectation?

Reply:
Good find. I accidentally tested the same case twice and only changed the comment.